hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/BulkUploadHandler.java

```diff
@@ -283,13 +287,16 @@ private void readEachLine(final String line, final double innerCounter, final do
             if (Math.abs(next - 0.1) < 0.00001 || current - next >= 0 || next - current >= 0.05
                     || Math.abs(next - 1) < 0.00001) {
                 managementUIState.getTargetTableFilters().getBulkUpload().setProgressBarCurrentValue(next);
-                managementUIState.getTargetTableFilters().getBulkUpload()
-                        .setSucessfulUploadCount(successfullTargetCount);
-                managementUIState.getTargetTableFilters().getBulkUpload().setFailedUploadCount(failedTargetCount);
-                eventBus.publish(this, new TargetTableEvent(TargetComponentEvent.BULK_TARGET_CREATED));
+                updateBulkUpload();
             }
         }
 
+        private void updateBulkUpload() {
+            managementUIState.getTargetTableFilters().getBulkUpload().setSucessfulUploadCount(successfullTargetCount);
+            managementUIState.getTargetTableFilters().getBulkUpload().setFailedUploadCount(failedTargetCount);
+            eventBus.publish(this, new TargetTableEvent(TargetComponentEvent.BULK_TARGET_CREATED));
```

Comment:
How do you now here, that a target was created. This will be called within a `finally`-block (see line 232)